src/clients/ETHClient.ts

```diff
@@ -335,22 +335,22 @@ export class ETHClient {
     );
     // logger("sendDeposit message", message)
 
-    let signatureResult:
-      | {
-          owner: string;
-          r: string;
-          s: string;
-          v: string;
-        }
-      | undefined;
+    // const signatureResult:
```

Comment:
Is this needed? Try not to leave commented code as it may be confusing to future developers.